backend/app/repository/gitee_provider.py

```
@@ -317,9 +317,14 @@ def validate_token(self, token: str, git_domain: str = None) -> Dict[str, Any]:
         # Use custom domain if provided, otherwise use default
         api_base_url = self._get_api_base_url(git_domain)
 
+        # Re-validating an already stored credential hands this the ciphertext, the
+        # same way the other providers are given it. Decrypting here keeps that case
+        # from reading as an invalid token.
+        decrypt_token = self.decrypt_token(token)
+
         try:
             response = requests.get(
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)